includes/Admin/ImgAreaSelect_Patch.php

```diff
@@ -36,6 +36,8 @@
 
 /**
  * Class ImgAreaSelect_Patch
+ *
+ * TODO remove this workaround https://core.trac.wordpress.org/ticket/54308 is committed.
```

Comment:
That one is already committed. So I guess you meant we can remove once WP 6.2 is the min requirement?